backend/src/api/podcasts/podcasts.api.ts

```diff
@@ -24,8 +25,8 @@ const initPodcastsApi = ({ apiRouter, podcastService }: Args): Router => {
 
   podcastRouter.get(
     PodcastsApiPath.ROOT,
-    handleAsyncApi(async (_req, res) => {
-      return res.json(await podcastService.getAll()).status(HttpCode.OK);
+    handleAsyncApi(async (req, res) => {
+      return res.json(await podcastService.getByQuery(req.query as unknown as PodcastLoadFilter)).status(HttpCode.OK);
```

Comment:
```suggestion
      return res.json(await podcastService.getByQuery(req.query as PodcastLoadFilter)).status(HttpCode.OK);
```

Reply:
If i remove "as unknown"  i have error
`Conversion of type 'ParsedQs' to type 'PodcastLoadFilter' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
  Type 'ParsedQs' is missing the following properties from type 'PodcastLoadFilter': offset, limit, searchts(2352)`